.buildkite/scripts/common.sh

```diff
@@ -53,7 +53,7 @@ getOSOptions() {
 # Wrapper function for executing mage
 mage() {
     go version
-    if ! [ -x "$(type -p mage | sed 's/mage is //g')" ];
+    if ! [ -x "$(type -P mage | sed 's/mage is //g')" ];
```

Comment:
so, because we define a `mage` bash function that encases the actual mage executable and we only want to check if the actual binary is installed, `type -p` seems to be the only command I found that behaves in the way the script initially expected. The previous version would always return true.